staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -447,6 +448,11 @@ func (r *crdHandler) getOrCreateServingInfoFor(crd *apiextensions.CustomResource
 		scaleSpec = crd.Spec.Subresources.Scale
 	}
 
+	table, err := tableconvertor.New(nil)
```

Comment:
// TODO: replace the nil with the actual extensions when we support them in OpenAPI validation specs.